snowpenguin/django/recaptcha2/templates/snowpenguin/recaptcha/recaptcha_init.html

```diff
@@ -1 +1,33 @@
-<script src="https://www.google.com/recaptcha/api.js{% if explicit %}?onload=djangoRecaptchaOnLoadCallback&render=explicit{% if language %}&hl={{ language }}{% endif %}{% elif language %}?hl={{ language }}{% endif %}" async defer></script>
\ No newline at end of file
+<script src="https://www.google.com/recaptcha/api.js{% if explicit %}?render=explicit{% if language %}&hl={{ language }}{% endif %}{% elif language %}?hl={{ language }}{% endif %}" async defer></script>
+{% if explicit %}
+<script>
+    window.addEventListener('load', function() {
+      var captchas = [].slice.call(document.querySelectorAll('.g-recaptcha'));
+
+      captchas.map(function(el) {
+        var containerID = el.id;
+        var options = {'sitekey': el.dataset.sitekey};
+
+        if (el.dataset.theme) {
```

Comment:
So to get around all the manual checking, we can just loop over all of the keys / values stored within`el.datasets`. Something like... 

```
Object.keys(el.datasets).forEach(key => {
  options[key] = el.datasets[key]
})
```

should work